### PR TITLE
keyboard shortcut to delete current playing file added

### PR DESCRIPTION
**Pull request by seran**
I've added a keyboard shortcut to delete current playing file. Personally I needed this. If you think this could be useful feature, please consider merging this.

### CODE REVIEW

**Comment by ftoledo** [comment]
i second this. please merge

also #28 is related

File: keys.h [hunk 1]
```diff
@@ -105,7 +105,8 @@ enum key_cmd
 	KEY_CMD_LYRICS,
 	KEY_CMD_QUEUE_TOGGLE_FILE,
 	KEY_CMD_QUEUE_CLEAR,
-	KEY_CMD_WRONG
+	KEY_CMD_WRONG,
+	KEY_CMD_DELETE_CURRENT_FILE
```
Comment (moreaki) [inline keys.h]: Don't you have to switch the order, so that KEY_CMD_WRONG is always the last entry?

Comment (seran) [reply keys.h]: I went through the source code, I don't see anywhere the order of the enum considered for anything. If there is a reason, I'm happy to change it.

**Comment by silvus** [comment]
A merge is unlikely, this is not an official repository, juste a mirror of the upstream subversion repo.
See [jsonsafari comment](https://github.com/jonsafari/mocp/pull/4#issuecomment-367900290) or the README.

**Pull request closed without merging**